lookout/style/typos/typos_functions.py

```diff
@@ -0,0 +1,176 @@
+import random
+import itertools
+
+import numpy
+import pandas
+
+
+def rand_bool(true_prob):
+    """
+    Returns True with probability true_prob
+    """
+    return random.uniform(0, 1) < true_prob
+
+
+def read_frequencies(file):
+    """
+    Read token frequencies from file.
+    """
+    frequencies = {}
+    with open(file, "r") as f:
+        for line in f:
+            split = line.split()
+            frequencies[split[0]] = int(split[1])
+    return frequencies
```

Comment:
we have `asdf` model with frequencies. I wonder, If you use that model, why do you read it not as a model file?